repos/exawind/packages/trilinos/package.py

```diff
@@ -17,6 +17,7 @@ class Trilinos(bTrilinos, CmakeExtension):
     # the trilinos github page appended with the date of the commit.
     # this preserves the Trilinos versioning scheme and will allow for valid
     # version comparisons in spack's internals.
+    version("15.1.1", tag="trilinos-release-15-1-1")
     version("14.1.0.2023.02.28", commit="8b3e2e1db4c7e07db13225c73057230c4814706f")
```

Comment:
move this back to `13.4.1`?

Reply:
I think we want to try to use the latest trilinos version with nalu though. I figured we could switch to actual releases.

Reply:
I agree with that. I just want to make sure we can build the challenge problem that ran across platforms for historical purposes.